hokusai/production.yml

```diff
@@ -173,4 +173,35 @@ spec:
                   - key: tier
                     operator: In
                     values:
-                    - background
\ No newline at end of file
+                      - background
+
+---
```

Comment:
I was under the impression we only want to run this in one environment, i.e. staging?

Reply:
Ah okay gotcha - I see the notes above to delete the staging job and point positron staging back at the staging db